src/index.html

```diff
@@ -10,23 +10,37 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar airbnb</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="big-search"
+      data-qa="big"
+    >
```

Comment:
Your deployment does not seem to be on the latest version of your code:

GH-Pages:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/52002856/5c5f4128-994d-457f-ad13-13be44e98c3e)